tests/Integration/setup.php

```diff
@@ -40,23 +48,27 @@ public function __construct($request)
         $this->request = $request;
     }
 
-    protected function parameter($name)
+    protected function parameter($name, $cookie = false)
```

Comment:
Can we split this into two functions (`parameter` and `cookieParameter`)? Otherwise the function is basically doing two things. Here's some interesting discussion on that topic in general: https://softwareengineering.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior

Reply:
I was actually wondering about it, good point, this discussion solves my conflict :-) good point